lib/index.js

```diff
@@ -74,10 +76,20 @@ export default function rehypePicture(options) {
 
       for (key in map) {
         if (Object.hasOwn(map, key)) {
+          /** @type {{srcSet: string, type: string | null | undefined, width?: string, height?: string}} */
```

Comment:
If you put a `@typedef {import('hast').Properties} Properties` at the top of the file, you can use that type here:

```suggestion
          /** @type {Properties} */
```